lib/class-sp-api.php

```diff
@@ -290,7 +290,12 @@ public function index_posts( $posts ) {
 				$body[] = addcslashes( $json, "\n" );
 			}
 		}
-		
+
+		// If no posts should be indexed, return an empty response.
+		if ( empty( $body ) ) {
+			return (object) [ 'items' => [] ];
```

Comment:
```suggestion
			return (object) array( 'items' => array() );
```

Is this even the right syntax? I already forget

Reply:
ah, yes, in that case, we should use `array()` 😂  it looks quite strange now.